File: LeetCode-Programs/Check If N and Its Double Exist.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        n=0
        for i in range(len(arr)):
            n=2*arr[i]
            for j in range(len(arr)):
                if j!=i and n==arr[j]:
                    return True
        return False

File: LeetCode-Programs/Check If N and Its Double Exist.cpp
class Solution {
public:
    bool checkIfExist(vector<int>& arr) {
        int n;
        for(int i=0;i<arr.size();i++)
        {
            n=2*arr[i];
            for(int j=0;j<arr.size();j++)
            {
                if(j!=i && n==arr[j])
                    return true;
            }
        }
        return false;
        
    }
};
